### ISSUE TITLE
[BUG] MediaElement crashes application on Windows if MetadataArtworkUrl is not set.

### ISSUE THREAD

**Issue by crone66** [CommunityToolkit/Maui#2150]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

MediaElement crashs entrie application if MetadataArtworkUrl is not set on windows.

### Expected Behavior

Metadata should be optional or the documentation should clearly state that MetadataArtworkUrl is required. Additionally some code examples in the documentation don't use MetadataArtworkUrl which clearly do not work right now.

### Steps To Reproduce

1. Download the Community toolkit sample.
2. Open the MediaElementPage.xaml
3. Remove MetadataArtworkUrl
4. Start the sample
5. open the media element view
6. crash

or Simply create a page with the following content and open it.
```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
             x:Class="MauiApp1.MainPage">

    <Grid RowDefinitions="*">
        <toolkit:MediaElement
            ShouldAutoPlay="True"
            Source="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"/>
    </Grid>

</ContentPage>

```

### Link to public reproduction project repository

https://github.com/CommunityToolkit/Maui

### Environment

```markdown
- .NET MAUI CommunityToolkit: 4.0.0 & 4.1.0
- OS: Windows 10 Pro 10.0.19045
- .NET MAUI: 8.0.80
```

**Comment by ne0rrmatrix** [comment]
This should be fixed with #2144 Can you try building nuget from source or try https://github.com/CommunityToolkit/Maui/wiki/Preview-Packages the preview package?

**Comment by crone66** [comment]
It's still not fixed. But At least it happens now after the debugger was attached and I get a nullreference exception now. Previously it just crashed before the debugger was attached and without any indication what was wrong. Therefore something was changed but it's not fixed yet.

I used the Latest prerelease 99.0.0-preview2585.
I get a Null-reference exception in the ExceptionHelpers.cs in L146 followed by a crash of the application.
